### PR TITLE
Fix: #AR-6268 auth-wagmi package SDK Reference Guide

### PR DESCRIPTION
**Pull request by shaloo**
`auth-wagmi` package auto-generated SDK Reference Guide updates.

### CODE REVIEW

File: typedoc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "entryPoints": ["src/index.ts"],
+  "name": "Auth SDK Reference Guide",
```
Comment (shaloo) [inline typedoc.json:3]:
```suggestion
"name": "Auth Wagmi SDK Reference Guide",

Comment (shaloo) [inline typedoc.json:3]:
```suggestion
"name": "Auth SDK Reference Guide",

File: typedoc.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "entryPoints": ["src/index.ts"],
+"name": "Auth SDK Reference Guide",
```
Comment (shaloo) [inline typedoc.json:3]:
```suggestion
 "name": "Auth SDK Reference Guide",

Comment (shaloo) [inline typedoc.json:3]:
```suggestion
  "name": "Auth Wagmi SDK Reference Guide",

File: typedochtml.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "entryPoints": ["src/index.ts"],
+  "name": "Auth SDK Reference Guide",
```
Comment (shaloo) [inline typedochtml.json:3]:
```suggestion
   "name": "Auth Wagmi SDK Reference Guide",

File: typedochtml.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "entryPoints": ["src/index.ts"],
+   "name": "Auth Wagmi SDK Reference Guide",
```
Comment (shaloo) [inline typedochtml.json:3]:
```suggestion
"name": "Auth Wagmi SDK Reference Guide",

**Comment by shaloo** [comment]
text+logo in sample updated as suggested by Aravindh and Sharan. 
Screenshot 2023-07-03 at 6 42 20 PM

**Review by makylfang — APPROVED** [review]

**Pull request merged**